src/User/Form/LoginForm.php

```diff
@@ -73,7 +73,7 @@ public function attributeLabels()
             'login' => Yii::t('usuario', 'Login'),
             'password' => Yii::t('usuario', 'Password'),
             'rememberMe' => Yii::t('usuario', 'Remember me next time'),
-            'twoFactorAuthenticationCode' => Yii::t('usuario', 'Two-factor authentication code')
+            'twoFactorAuthenticationCode' => Yii::t('usuario', '2FA code')
```

Comment:
I am not sure if 2FA is more descriptive than Two-Factor|Two Factor authentication.

Reply:
I know that I have agreed to it, but... not seems to brief. What do you think? We need to think as an external user.

Reply:
I could agree with you, but generally I think that people who wants to enable 2FA knows what it is and have a bit of knowledge about its acronym. Also in the account settings form I've left the extended and compact strings together, so ppl can associate 2FA to two factor auth.

But if you prefer I can change the label. Is it just about that or elsewhere?

Reply:
Is everywhere, I think two factor authentication is more descriptive not for developers but for end users. Not many people know (in our case we always have to explain what it is and how it works on all our panels), so the acronym even though is clearly a plus for developers I am not really sure about end users.

Reply:
made another commit